modules/farm_fd2_school/src/entrypoints/FD4_Application/harvest.e2e.cy.js

```diff
@@ -12,7 +12,58 @@ describe('Tests for the Harvest form', () => {
     cy.saveSessionStorage();
   });
 
-  it('Placeholder test', () => {
-    cy.get('[data-cy="does-not-exist"]');
+  it('Initial state of the Harvest form is correct', () => {
+    cy.get('[data-cy="harvest-header"]')
+      .should('be.visible')
+      .and('contain', 'Harvest');
+
+    cy.get('[data-cy="harvest-date"]')
+      .should('be.visible')
+      .find('input')
+      .should('have.value', '2019-06-15');
+
+    cy.get('[data-cy="harvest-crop"]')
+      .should('be.visible')
+      .find('select')
+      .should('have.value', null);
+
+    cy.get('[data-cy="harvest-buttons"]').should('be.visible');
+    cy.get('[data-cy="harvest-buttons"]')
+      .find('button')
+      .first()
+      .should('be.disabled');
+
+    cy.get('[data-cy="harvest-table"]').should('not.exist');
+    cy.get('[data-cy="harvest-quantity"]').should('not.exist');
+    cy.get('[data-cy="harvest-units"]').should('not.exist');
+    cy.get('[data-cy="harvest-comment"]').should('not.exist');
+  });
+
+  it('crop with harvestable plant', () => {
+    cy.get('[data-cy="harvest-crop"] select').select('BROCCOLI');
+    cy.get('[data-cy="harvest-table"]').should('exist').and('be.visible');
+
+    cy.get('[data-cy="harvest-quantity"] input')
```

Comment:
It would be better to use the `data-cy` value in the `find` to get the element inside of this component.  See the documentation for the component to find the `data-cy` values that it contains.